openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -153,29 +157,14 @@ def refresh_jwt_cookies(request, response):
             refresh_token = request.COOKIES[jwt_cookies.jwt_refresh_cookie_name()]
         except KeyError:
             raise AuthFailedError(u"JWT Refresh Cookie not found in request.")
-        _create_and_set_jwt_cookies(response, request, refresh_token=refresh_token)
-    return response
-
-
-def _set_deprecated_logged_in_cookie(response, request):
```

Comment:
**Note to reviewers:** This function was simply moved below.  I removed the `_` from the name of the function below (`set_deprecated_user_info_cookie`) since it is used externally.